code/modules/surgery/organs/heart.dm

```diff
@@ -427,6 +427,7 @@
 	resistance_flags = FIRE_PROOF
 	density = TRUE
 	anchored = TRUE
+	flags_ricochet = RICOCHET_SHINY | RICOCHET_HARD
```

Comment:
RICOCHET_HARD means it reflects bullets

Reply:
The base structure has RICOCHET_HARD too so it is just keeping old behavior.